nautobot/extras/migrations/0036_auto_20220721_1356.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.14 on 2022-07-21 18:56
```

Comment:
Please delete these migrations and recreate them so that there is only a single migration file.